test/node.js

```diff
@@ -14,4 +14,11 @@ describe('verify-dist', async () => {
    */
     await expect(access(resolve(cwd, '../dist/ipfs-sw-sw.js'), constants.F_OK)).to.not.be.rejected()
   })
+
+  it('has css file with expected content', async () => {
+    await expect(access(resolve(cwd, '../dist/ipfs-sw-styles.css'), constants.F_OK)).to.not.be.rejected()
+    const contents = await readFile(resolve(cwd, '../dist/ipfs-sw-styles.css'), 'utf8')
+    expect(contents).to.include('47vh')
+    expect(contents).to.include('.local-storage-toggle input.status')
+  })
```

Comment:
we will likely want to remove this in the future but i'm going to leave it for now